### ISSUE TITLE
No return / No response

### ISSUE THREAD

**Issue by ghost** [Gokul1503A/Siri-Grok-Integration#1]
Gokul,
Not sure if I'm using the shortcut wrong but I injected my API that I have copied from the Generate API Key off xAI Cloud Console and when I run the shortcut, there is no response when asking a question after activating "Ask Grok"

**Comment by ghost** [comment]
Brother!

I found the problem

We needed set the "base url" to https://api.x.ai/v1

![image](https://github.com/user-attachments/assets/6cc2574b-b3e2-41d9-a911-84a09e016b9c)

**Comment by Gokul1503A** [comment]
I think the issue might be in the authorization section. You should add ‘Bearer ‘ before your API key. I’ve attached an image to show you what I mean. 

![IMG_4338D3B89054-1](https://github.com/user-attachments/assets/ab2b6ccf-717e-4972-b549-02e8c09f0e08)

**Comment by ghost** [comment]
Yes.  That was one part of the issue.  I've included Bearer in the code but also needed to add base url:  with the base url of X.ai
